mobilecoind/src/config.rs

```diff
@@ -166,26 +169,27 @@ impl Config {
 
     /// Get the attestation verifier used to verify fog reports when sending to
     /// fog recipients
-    pub fn get_fog_ingest_verifier(&self) -> Option<Verifier> {
-        self.fog_ingest_enclave_css.as_ref().map(|signature| {
-            let mr_signer_verifier = {
-                let mut mr_signer_verifier = MrSignerVerifier::new(
-                    signature.mrsigner().into(),
-                    signature.product_id(),
-                    signature.version(),
-                );
-                mr_signer_verifier.allow_hardening_advisories(&[
-                    "INTEL-SA-00334",
-                    "INTEL-SA-00615",
-                    "INTEL-SA-00657",
-                ]);
```

Comment:
I think this earlier code was somewhat wrong anyways, there is a better conversion like `MrSignerVerifier::from(..._sigstruct())` that was being used elsewhere, and the hardening advisories are now in a measurement crate list